code/game/machinery/computer/abnormality_work.dm

```diff
@@ -243,11 +248,14 @@
 		else
 			visible_message("<span class='notice'>Work Result: Bad</span>")
 	if(istype(user))
-		if(!training)
+		if(training)
+			if(tutorial)
+				datum_reference.work_complete(user, work_type, pe, work_speed*datum_reference.max_boxes, was_melting, canceled, tutorial)
+			else //only called by training rabbit, skips stat gain
+				datum_reference.current.WorkComplete(user, work_type, pe, work_speed*datum_reference.max_boxes)
```

Comment:
You can just combine it and pass the tutorial as an argument, since you only edit this thing.